nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -206,6 +206,11 @@ language governing permissions and limitations under the License. -->
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.xerial.snappy</groupId>
+            <artifactId>snappy-java</artifactId>
```

Comment:
Doing a cursory scan, everything looks pretty good, although I notice that this dependency was not added into the NOTICE file.  Would imagine that would be needed, but am certainly not the licensing expert. 

@joewitt NOTICE sufficient, or does this need to be elsewhere as well?